src/open_inwoner/utils/forms.py

```diff
@@ -6,6 +6,44 @@
 from django.utils.translation import gettext_lazy as _
 
 
+class ErrorMessageMixin:
+    custom_error_messages = {
+        "required": _(
+            "Het verplichte veld %s is niet (goed) ingevuld. Vul het veld in."
+        )
+    }
+
+    def __init__(self, *args, **kwargs):
+        user_added_messages = {}
```

Comment:
This should be simply `custom_error_messages`, it contrasts with the class-level `default_error_messages`. This way the code is consistent with the test you wrote.